grammars/prql-lezer/src/prql.grammar

```diff
@@ -74,7 +74,7 @@ Op_bin { op_bin_only | !term op_unary }
   line_wrap { "\\" }
   wrapped_line { newline+ (Comment newline+)* line_wrap }
   newline { "\n" }
-  Range { ".." }
+  Range { @digit+ ".." @digit+ }
```

Comment:
Can we add a TODO?

```suggestion
  // TODO: Because this can also be used to compile to BETWEEN, ranges should
  // allow any literal, and arguably any expression. And if possible it shouldn't allow for spaces.
  Range { @digit+ ".." @digit+ }
```